androidtest/src/main/java/org/odk/collect/androidtest/LiveDataTestUtils.kt

```diff
@@ -22,19 +22,18 @@ fun <T> LiveData<T>.getOrAwaitValue(
 ): T {
     var data: T? = null
     val latch = CountDownLatch(1)
-    val observer = object : Observer<T> {
-        override fun onChanged(o: T) {
-            data = o
-            latch.countDown()
-            this@getOrAwaitValue.removeObserver(this)
```

Comment:
This didn't work with cases where there was already a value set on the `LiveData`.